addons/account/models/account_tax.py

```diff
@@ -552,90 +552,688 @@ def onchange_price_include(self):
         if self.price_include:
             self.include_base_amount = True
 
-    def _compute_amount(self, base_amount, price_unit, quantity=1.0, product=None, partner=None, fixed_multiplicator=1):
-        """ Returns the amount of a single tax. base_amount is the actual amount on which the tax is applied, which is
-            price_unit * quantity eventually affected by previous taxes (if tax is include_base_amount XOR price_include)
-        """
+    def _prepare_dict_for_taxes_computation(self):
         self.ensure_one()
+        tax_values = {
+            'tax_id': self.id,
+            'tax__name': self.name,
+            'tax__amount_type': self.amount_type,
+            'tax__sequence': self.sequence,
+            'tax__amount': self.amount,
```

Comment:
double _ because the entry `tax_amount` is already taken a bit everywhere. To not track all those values, I added the double _ to say it's a field from the tax... :thinking: I can try to remove the prefix `tax__`

Reply:
I think I saw some places in the JS where we expect both 'amount' and 'tax__amount' to be accessed :see_no_evil: 

I'm not against keeping things like that, I just find it weird :)